test/message_bus/message_inbox/setup_message_inbox.js

```diff
@@ -0,0 +1,183 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const BN = require('bn.js');
+const web3 = require('../../test_lib/web3.js');
+const Utils = require('../../test_lib/utils.js');
+const { AccountProvider } = require('../../test_lib/utils.js');
+
+const MessageInbox = artifacts.require('MessageInboxDouble');
+
+contract('MessageInbox::setupMessageInbox', (accounts) => {
+  let messageInbox;
+  const setupParams = {};
+  const accountProvider = new AccountProvider(accounts);
+
+  beforeEach(async () => {
+    setupParams.metachainId = Utils.generateRandomMetachainId();
+    setupParams.messageOutbox = await accountProvider.get();
+    setupParams.outboxStorageIndex = new BN(Utils.getRandomNumber(63));// need to verify thr range
```

Comment:
```
// need to verify thr range
```
What does it mean here?

Reply:
Range is verified.

Reply:
You can remove the comment. I think it's not needed.